proxy.conf.json

```diff
@@ -1,6 +1,11 @@
 {
-    "/api": {
-        "target": "http://localhost:7001",
-        "secure": false
+    "/api/graphql": {
```

Comment:
mmh..looks like this conf here is not used, you configure the service in app.module.ts